whisper_server/whisper_api_server.py

```
@@ -211,40 +212,35 @@ def get_model(model_name: str):
         try:
             pipe.model = torch.compile(pipe.model, mode="reduce-overhead")
             logger.info("Torch compile enabled for model optimization")
         except Exception as e:
             logger.warning(f"Torch compile failed: {str(e)}, continuing without it")
-            
+
         logger.info(f"Model {local_model_name} loaded successfully with optimizations")
         return pipe, batch_size
-        
+
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)